removeDuplicates.js

```diff
@@ -0,0 +1 @@
+const uniqueArray = array => [...new Set(array)]
```

Comment:
Is this function finding a unique array or array of unique items?
Try to rename it in the way others can get what it does without confusion.